content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -9,6 +9,12 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - 2020-06-30 - Developer preview
```

Comment:
I was wrong, we don't need the date in this line, sorry.

Reply:
No prob, removed